lib/src/tools/over_react_format_tool.dart

```diff
@@ -0,0 +1,49 @@
+import 'dart:async';
+import 'dart:io';
+
+import 'package:dart_dev/dart_dev.dart';
+import 'package:dart_dev/utils.dart';
+import 'package:io/io.dart';
+
+import '../tools/format_tool.dart';
+
+class OverReactFormatTool extends DevTool {
+  /// Wrap lines longer than this.
+  ///
+  /// Default is 80.
+  int lineLength;
+
+  @override
+  String description =
+      'Format dart files in this package with over_react_format.';
+
+  @override
+  FutureOr<int> run([DevToolExecutionContext context]) async {
+    Iterable<String> paths = context?.argResults?.rest;
+    if (paths?.isEmpty ?? true) {
+      if (context != null) {
+        context.usageException(
+            '"hackFastFormat" must specify targets to format.\n'
+            'hackFastFormat should only be used to format specific files. '
+            'Running the command over an entire project may format files that '
+            'would be excluded using the standard "format" command.');
+      } else {
+        logError(
+            '"hackFastFormat" must specify targets to format.\n',
+            'hackFastFormat should only be used to format specific files. '
+                'Running the command over an entire project may format files that '
+                'would be excluded using the standard "format" command.');
+        return ExitCode.usage.code;
+      }
```

Comment:
`context` shouldn't ever be null when actually running via `pub run dart_dev`, so is this branch just for unit tests? If that's the case, I'd say we get rid of this (and `logError()`) and just make sure we pass in a `DevToolExecutionContext` argument when we want to test the above branch.

Reply:
👍  Sweeet, that makes things easier. I originally had a fallback because `run` only takes in context as an optional parameter, so I didn't want to not include a case for that. If we're comfortable with it though, it's much nicer to assume it will be there